src/HDInsight/HDInsight/help/New-AzHDInsightCluster.md

```diff
@@ -1004,6 +1023,22 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -PrivateLink
+Gets or sets the private link type.
+
+```yaml
+Type: System.String
+Parameter Sets: (All)
+Aliases:
+Accepted values: Enabled, Disabled
```

Comment:
If `-PrivateLink` can only be enabled or disabled, how about we use a switch parameter `[-EnablePrivateLink]` (or disable) instead?

Reply:
> If `-PrivateLink` can only be enabled or disabled, how about we use a switch parameter `[-EnablePrivateLink]` (or disable) instead?

Hi @isra-fel , Thanks for your comment. I understand your point. This is because RP is using string type instead of bool, I think that I should keep consistent with RP in case someday I need to change the parameter's type and bring breaking change. And RP think using string is a declarative way. 
How do you like?